fancy_prompt/fancy_prompt.py

```diff
@@ -109,14 +109,14 @@ def undodb_get_time():
     return udb.time.get()
 
 
-TERM_BRIGHT_WHITE = r'\[\033[1m\]'
-TERM_BRIGHT_YELLOW = r'\[\033[1;33m\]'
-TERM_BRIGHT_GREEN = r'\[\033[1;32m\]'
-TERM_BRIGHT_RED = r'\[\033[1;31m\]'
-TERM_BRIGHT_BLUE = r'\[\033[1;34m\]'
-TERM_MAGENTA = r'\[\033[35m\]'
-TERM_CYAN = r'\[\033[36m\]'
-TERM_RESET = r'\[\033[m\]'
+TERM_BRIGHT_WHITE = '\\[\033[1m\\]'
+TERM_BRIGHT_YELLOW = '\\[\033[1;33m\\]'
+TERM_BRIGHT_GREEN = '\\[\033[1;32m\\]'
+TERM_BRIGHT_RED = '\\[\033[1;31m\\]'
+TERM_BRIGHT_BLUE = '\\[\033[1;34m\\]'
+TERM_MAGENTA = '\\[\033[35m\\]'
+TERM_CYAN = '\\[\033[36m\\]'
+TERM_RESET = '\\[\033[m\\]'
```

Comment:
Now that you use `gdb.prompt.substitute_prompt`, what happens if you revert this change?

Reply:
The original version with raw strings still doesn't work. Without raw strings it works but then pylint complains about 'Anomalous backslash in string'.